src/Data/Error.hs

```diff
@@ -59,7 +60,7 @@ formatError includeSource colourize blob@Blob{..} Error{..}
   . (if Flag.toBool LogPrintSource includeSource then showExcerpt colourize errorSpan blob else id)
   . showCallStack colourize callStack . showChar '\n'
   where
-    path = Just $ if Flag.toBool LogPrintSource includeSource then blobPath blob else "<filtered>"
+    path = Just $ if Flag.toBool LogPrintSource includeSource then Path.toString $ blobPath blob else "<filtered>"
```

Comment:
`blobFilePath` here?

Reply:
Yes, it shall be.  I will fix it in next commit.